assets/wire/components/ItemStatisticsModal.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Modal from 'components/Modal';
+import PropTypes from 'prop-types';
+
+export default class ItemStatisticsModal extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {loading: true, iframe: null};
+        this.fetchState();
```

Comment:
`fetchState` should be placed in `componentDidMount` lifecycle or load the state before rendering this component